contrib/junit.tpl

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+<testsuites>
+{{ range . }}
+	{{- $failures := 0 }}
+    {{- range .Vulnerabilities -}}
+	{{- $failures = sum $failures 2 -}}
+	{{- end -}}
+	<testsuite tests="1" failures="{{ $failures }}" time="" name="{{  .Target }}">
+		{{- if not (eq .Type "") }}
+        <properties>
+            <property name="type" value="{{ .Type }}"></property>
+        </properties>
+        {{- end -}}
+        {{ range .Vulnerabilities }}
+        <testcase classname="{{ .PkgName }}-{{ .InstalledVersion }}" name="{{ .VulnerabilityID }}" time="">
+            <failure message={{ .Title | printf "%q" }} type="description">{{ .Description | printf "%q" }}</failure>
+            <failure message="" type="severity">{{ .Vulnerability.Severity }}</failure>
```

Comment:
After the test on CircleCI, I found CircleCi didn't show the second `failure`.

![image](https://user-images.githubusercontent.com/2253692/85728391-8952a800-b700-11ea-8541-a27d25562658.png)

We probably should embed severity into `name` in `testcase` like 
```
"【{{ .Vulnerability.Severity }}】 {{ .VulnerabilityID }}"

Reply:
@knqyf263 done